DQMServices/Core/src/DQMEDAnalyzer.cc

```diff
@@ -0,0 +1,53 @@
+#include "DQMServices/Core/interface/DQMEDAnalyzer.h"
+#include "DataFormats/Histograms/interface/DQMToken.h"
+
+DQMEDAnalyzer::DQMEDAnalyzer() {
+  produces<DQMToken,edm::Transition::EndLuminosityBlock>();
+  produces<DQMRunToken,edm::Transition::EndRun>();
```

Comment:
@Dr15Jones when I remove the `Run` in this line, I got an framework error, saying I cannot have more than on produce call for the same product. I guess I have to specify both transitions in one call?

Reply:
Hmm. Our framework test works: 
https://github.com/cms-sw/cmssw/blob/ede02fc7b4740a09bb6e578d9ec71129d8c19b19/FWCore/Integration/test/ThingProducer.cc#L14

However, that does use unique product instance names, which was not meant to be required.